build-system/dep-check-config.js

```diff
@@ -321,6 +321,8 @@ exports.rules = [
       'src/polyfills.js->src/polyfills/object-assign.js',
       'src/polyfills.js->src/polyfills/promise.js',
       'src/polyfills.js->src/polyfills/array-includes.js',
+      'src/polyfills.js->src/polyfills/array-from.js',
```

Comment:
I don't see array-from anywhere.

Reply:
Ah sorry just removed it and missed this line. Thanks for catching